### DISCUSSION TITLE
What is FXIOS-6036?

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#15508] [Q&A]
I see a lot of references to it in the source that various bits of code will be removed with FXIOS-6036, but because I don't have access to Mozilla's Jira instance I've got no idea what it actually is.

My fork has bits of code that will probably need updating when it lands so it would be nice to have some prior info to prepare with.

**Comment by lmarceau** [comment]
Hello! Thanks for asking, this refers to https://github.com/mozilla-mobile/firefox-ios/issues/13685. This task will be to clean up the old codepath when `CoordinatorFlagManager.isCoordinatorEnabled` is false.
